sqlalchemy_cockroachdb/base.py

```diff
@@ -150,14 +150,15 @@ def initialize(self, connection):
         if self._is_v211plus and not self.disable_cockroachdb_telemetry:
             dialect_version = pkg_resources.require("sqlalchemy-cockroachdb")[0].version
             sqlalchemy_version = pkg_resources.require("sqlalchemy")[0].version
+            sqlalchemy_version_series = re.match("(.+)\.", sqlalchemy_version).group(1)
```

Comment:
i think it might be better to make the regex more clear, like:
```
([0-9]+\.[0-9]+).*
```

as it is, a version like `1.2.3.4` would return `1.2.3` for the version series, which isn't what we'd want.

also perhaps add a fallback case so that if the regex doesn't match, then we just use `sqlalchemy_version` as is